### PR TITLE
chore: #6772 july housekeeping

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #6772

### CODE REVIEW

File: packages/elements/package.json [hunk 1]
```diff
@@ -71,36 +71,36 @@
...
+    "eslint": "8.20.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "jest": "^28.1.2",
+    "jest": "^28.1.3",
     "jest-coverage-badges": "^1.1.2",
     "move-file-cli": "^2.0.0",
     "node-polyfill-webpack-plugin": "^1.1.4",
     "postcss": "^8.4.14",
-    "prettier": "2.2.1",
+    "prettier": "2.7.1",
```
Comment (willmcvay) [inline packages/elements/package.json:89]: Can't be updated, breaks Storybook, hence tied to an absolute version

File: packages/elements/package.json [hunk 2]
```diff
@@ -71,36 +71,36 @@
...
     "serverless": "^2.72.3",
     "serverless-s3-deploy": "^0.10.1",
     "serverless-s3-remover": "^0.6.0",
     "serverless-single-page-app-plugin": "^1.0.4",
-    "snyk": "^1.962.0",
+    "snyk": "^1.980.0",
     "tslib": "^2.4.0",
-    "typescript": "4.6.3",
-    "webpack": "^5.73.0"
+    "typescript": "4.7.4",
```
Comment (willmcvay) [inline packages/elements/package.json:102]: Can't be updated, breaks Storybook, hence tied to an absolute version

File: packages/elements-legacy/package.json [hunk 3]
```diff
@@ -96,33 +96,33 @@
...
     "rollup-plugin-css-only": "^3.1.0",
     "rollup-plugin-scss": "^3.0.0",
     "serverless": "^2.72.3",
     "serverless-s3-deploy": "^0.10.1",
     "serverless-s3-remover": "^0.6.0",
     "serverless-single-page-app-plugin": "^1.0.4",
     "tslib": "^2.4.0",
-    "typescript": "4.3.5",
-    "webpack": "^5.73.0"
+    "typescript": "4.7.4",
```
Comment (willmcvay) [inline packages/elements-legacy/package.json:124]: Can't be updated, breaks Storybook, hence tied to an absolute version

File: packages/elements-legacy/package.json [hunk 4]
```diff
@@ -96,33 +96,33 @@
...
+    "eslint": "8.20.0",
     "eslint-plugin-prettier": "^4.2.1",
     "fs-extra": "^9.1.0",
-    "jest": "^28.1.2",
+    "jest": "^28.1.3",
     "jest-coverage-badges": "^1.1.2",
     "node-polyfill-webpack-plugin": "^1.1.4",
     "postcss": "^8.4.14",
-    "prettier": "2.2.1",
+    "prettier": "2.7.1",
```
Comment (willmcvay) [inline packages/elements-legacy/package.json:113]: Can't be updated, breaks Storybook, hence tied to an absolute version

**Review by willmcvay — COMMENTED** [review]
Bugs commented on with Storybook may have been fixed but if not, I would revert the update versions to the previous fixed versions

File: packages/elements-legacy/package.json (hunk 3 above)
Comment (bashleigh) [reply packages/elements-legacy/package.json:124]: ma bad, didn't release it was elements-legacy. Doesn't specify in the prompt

**Review by willmcvay — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**